src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1779,27 +1856,39 @@ subroutine apply_calving_velocity(meshPool, geometryPool, velocityPool, domain,
       if (globalInfo(3) / (globalInfo(4) + 1.0e-30_RKIND) > 0.01_RKIND) then
          ! If stranded ice deletion results in more than a small amount of the total calving flux, this routine will not
          ! be accurate.  If this error gets triggered, either try using a smaller timstep or this routine needs improving.
-         call mpas_log_write("Deleting stranded floating cells accounts for more than 1% of calving loss." // &
-                 "  Try using a smaller timestep or apply_calving_velocity may need improvements for this simulation.")
+         call mpas_log_write("Deleting stranded floating cells accounts for more than 1% of ablation loss." // &
+                 "  Try using a smaller timestep or apply_front_ablation_velocity may need improvements for this simulation.")
          err_tmp = 1
          err = ior(err, err_tmp)
       endif
 
 
-      ! Update halos on calvingThickness before applying it.
+      ! Update halos on calvingThickness or meltingThickness before applying it.
```

Comment:
If we  modify the calling signature as I suggested above, then I think this halo update will have to be performed by the calling routine.  So this halo update should be deleted and there should be a note added to the subroutine description that a halo update should be made after this routine.

Reply:
Halo updates now in calling routines.